server/core_storage.go

```diff
@@ -458,9 +458,6 @@ WHERE
 }
 
 func StorageWriteObjects(ctx context.Context, logger *zap.Logger, db *sql.DB, authoritativeWrite bool, ops StorageOpWrites) (*api.StorageObjectAcks, codes.Code, error) {
-	// Ensure writes are processed in a consistent order.
-	sort.Sort(ops)
```

Comment:
@zyro, @ftkg , sorting prevents deadlocks. I suspect it also somehow helped CockroachDB in clustered mode.

Reply:
Good call, fixed and updated comments to ensure we don't forget.